freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/PasswordService.java

```diff
@@ -91,21 +91,20 @@ private void asyncSetPasswords(String operationId, String accountId, String acto
                 requests.add(triggerSetPassword(stack, stack.getEnvironmentCrn(), userId, password));
             }
 
-            List<SuccessDetails> success = new ArrayList<>();
-            List<FailureDetails> failure = new ArrayList<>();
             for (SetPasswordRequest request : requests) {
                 try {
                     waitSetPassword(request);
```

Comment:
this doesn't really update the sync status on completion of each environment because waitSetPassword is called serially. Update for completed environments can be blocked on the waitSetPassword of an environment that it still running.
Moving the status update to where the work is actually done will be a little tricky because the password is being set by the SetPasswordHandler and triggered through the EventBus.